opcua/server/history.py

```diff
@@ -162,7 +163,7 @@ def _create_subscription(self, handler):
         params.Priority = 0
         return Subscription(self.iserver.isession, params, handler)
 
-    def historize(self, node, period=timedelta(days=7), count=0):
+    def historize_var(self, node, period=timedelta(days=7), count=0):
```

Comment:
histprize_data_change would be a better name since this is what we use at other places